static/app/views/explore/logs/utils.tsx

```diff
@@ -99,3 +122,78 @@ export function getLogBodySearchTerms(search: MutableSearch): string[] {
   }
   return searchTerms;
 }
+
+export function logsFieldAlignment(...args: Parameters<typeof fieldAlignment>) {
+  const field = args[0];
+  if (field === OurLogKnownFieldKey.TIMESTAMP) {
+    return 'right';
+  }
+  return fieldAlignment(...args);
+}
+
+export function removeSentryPrefix(key: string) {
+  return key.replace('sentry.', '');
+}
+
+export function getTableHeaderLabel(field: OurLogFieldKey) {
+  return LogAttributesHumanLabel[field] ?? removeSentryPrefix(field);
+}
+
+export function isLogAttributeUnit(unit: string | null): unit is LogAttributeUnits {
+  return (
+    unit === null ||
+    unit === `${DurationUnit}` ||
+    unit === `${CurrencyUnit}` ||
```

Comment:
@k-fish can you explain how this is intended to work please?

I’m updating some lint rules and getting a violation here because stringifying `DurationUtils` with `${DurationUnit}` just yields the string `'[object Object]'`, as `DurationUnit` is an enum.

If the plan is to check if the string `unit` is part of the enum `DurationUnit`, I think we would need:

```
Object.values(DurationUnit).includes(unit)
```

If this is the intended check, I can update it to this version in my PR.

Reply:
Yeah, this is a guard, so the intended check is against the enum, I'm not sure where I got this syntax from. Feel free to update

Reply:
PR here:
- https://github.com/getsentry/sentry/pull/88913